### ISSUE TITLE
Implement postcss-resemble-image for the header

### ISSUE THREAD

**Issue by ben-eb** [postcss/postcss.org#222]
Would be good to have this in case of a slow internet connection, it's a bit more interesting than a solid colour;

https://github.com/ben-eb/postcss-resemble-image

Original article: http://csswizardry.com/2016/10/improving-perceived-performance-with-multiple-background-images/

**Comment by ai** [comment]
More PostCSS plugins in PostCSS website! :)

/cc @marcustisater

**Comment by marcustisater** [comment]
Yeah, I'm sorry 😢  I have been super busy with planning [WordCamp](2016.stockholm.wordcamp.org), I promise to look it through and sort of some stuff by the end of the month. We need to get the basic pages up as well so we can start with #26

**Comment by marcustisater** [comment]
I will have a look at this tomorrow and also update pheonomic.
